lib/Conversion/TorchToTcp/TcpCustomOp.cpp

```diff
@@ -211,6 +214,118 @@ class ConvertAtenFakeQuantizePerChannelAffineOp
   }
 };
 
+class ConvertAtenSortOp : public OpConversionPattern<AtenSortOp> {
```

Comment:
Most likely the sort is coming from a top-k.  I would prefer that we keep top-k instead of changing it to a sort as trt supports top-k already, and if we add sort then it just ends up getting changed back to top-k internally with additional performance penalties due to any extra ops that get added

Reply:
Good idea. We can add an internal patch [here](https://sourcegraph.robot.car/github.robot.car/cruise/mla-robocomp-torch-mlir/-/blob/lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp?L8848) to disable the topk-->sort. I will add a new custom_op to handle the topk then.